pkgs/development/libraries/libuecc/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit, pkgs }:
+
+stdenv.mkDerivation rec {
+  version = "v5";
+  name = "libuecc-${version}";
+
+  src = fetchgit {
+    url = "git://git.universe-factory.net/libuecc";
+    rev = "refs/tags/${version}";
+    sha256 = "1dac165cfd7e4ebe38fca6da256e3816a21f0c2dbe2f515ddb185f76687dabd5";
+  };
+
+  buildInputs = [];
+
+  nativeBuildInputs = [ pkgs.cmake pkgs.pkgconfig pkgs.doxygen ];
```

Comment:
Of course, the dependencies should be passed via callPackage not `pkgs`, good catch